### PR TITLE
BoardMember Services added

### PR DESCRIPTION
**Pull request by nisalrenuja**

### CODE REVIEW

File: src/api/services/BoardMember.service.ts [hunk 1]
```diff
@@ -1,25 +1,91 @@
...
+};
 /**
  * @todo create @function deleteBoardMemberDetails to delete member
  * @param boardMemberId @type string
  */
+export const deleteBoardMemberDetails = async (boardMemberId: string) => {
+  return await BoardMemberModel.findById(boardMemberId)
+    .then(async (boardMemberDetails) => {
+      if (boardMemberDetails) {
+        return await boardMemberDetails.save();
```
Comment (senuravihanjayadeva) [inline src/api/services/BoardMember.service.ts:83]: should be updated deletedAt attribute

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

File: src/api/services/BoardMember.service.ts (hunk 1 above)
Comment (senuravihanjayadeva) [reply src/api/services/BoardMember.service.ts:83]: In the IBoardMember and IExecutiveBoard interfaces deletedAt attribute is missing. Please add that also. ( Also add those to the BoardMember and ExecutiveBoard Models ) You can refer IEvent interface and Event Model.

**Review by rusiruavb — CHANGES_REQUESTED** [review]
Make the changes that @senuravihanjayadeva have made.

**Comment by nisalrenuja** [comment]
> Make the changes that @senuravihanjayadeva have made.

I did the changes that @senuravihanjayadeva have requested

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**